app/src/main/res/layout/concept_card_fragment.xml

```diff
@@ -51,9 +51,9 @@
             android:id="@+id/concept_card_heading_text"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginStart="28dp"
```

Comment:
On line `42` use `general_screen_padding_bottom` instead of `108dp`.

```
      <ScrollView
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:overScrollMode="never"
        android:paddingBottom="@dimen/general_screen_padding_bottom"
        android:scrollbars="none">
```